.gitignore

```
@@ -1,3 +1,2 @@
-EleksTubeHAX/global_defines.h
+EleksTubeHAX/secrets.h
```

Just noticed this.  You're .gitignoring `secrets.h` but the file is called `SECRETES.h`.

I'm guessing you're on a computer that uses a case insensitive filesystem, aren't you?  :-)

Please keep file things case-sensitive.  Assume your users are using case sensitive filesystems.
```suggestion
EleksTubeHAX/SECRETS.h
```

---

secrets.h (and all capitalization variances :-) ) removed